### DISCUSSION TITLE
How to deploy webhook?

### DISCUSSION THREAD

**Discussion by juankstylez** [nutgram/nutgram#303] [General]
I am trying to deploy webhook ! Can you give me an example please? I already made the route in api.php

I know I must have ssl on my domain. That's how I'm implementing it.

What is all I have to configure? I don't understand! I have the following error!

https://prnt.sc/O0vRSsa3Oopr

Thank you !

**Comment by Lukasss93** [comment]
Give us the script you wrote for the webhook route please

**Reply by juankstylez** [reply]
This bro -> https://prnt.sc/jkirtYMqQb6r

`<?php

namespace App\Http\Controllers;

use Illuminate\Http\Request;
use SergiX44\Nutgram\Nutgram;

class FrontController extends Controller
{
    /**
     * Handle the telegram webhook request.
     */
    public function __invoke(Nutgram $bot)
    {
        $bot->run();
    }
}
`

**Comment by Lukasss93** [comment]
you don't have to call that route, telegram will do it after you set the webhook in this way:

```
php artisan nutgram:hook:set https://yourdomain.yeah/api/webhook
```

**Reply by juankstylez** [reply]
Ok I'll try !
